### ISSUE TITLE
lambda not pushing logs to cloudwatch log

### ISSUE THREAD

**Issue by Black742** [localstack/localstack#1401]
Does localstack support pushing lambda logs to Cloudwatch logs?

**Comment by anthonydelarosa** [comment]
that's exactly what I would like to know, when I run _$ awslocal logs describe-log-streams_ it asks me for the _--log-group-name_ but I have no idea what that is 'cause even though localstack Lambda is running, there is GUI so how can I find out the name so I can display it in the terminal

**Comment by ciaranevans** [comment]
I'm also having this issue in Python 3.6.8 using boto3.

Every <name>_c is a client from boto3.client() using the localstack url and port.

Example lambda:
```
import os
import logging

logger = logging.getLogger()
logger.setLevel(logging.INFO)

def handler(event, content):
  logger.info('Hello, World!')
```

Code to create execution role:
```
lambda_execution_role = iam_c.create_role(
        RoleName='LambdaExecutionRole'+str(datetime.datetime.now()),
        AssumeRolePolicyDocument='''{
        "Version": "2012-10-17",
        "Statement": [
            {
                "Effect": "Allow",
                "Action": [
                    "logs:*"
                ],
                "Resource": "arn:aws:logs:*:*:*"
            },
            {
                "Effect": "Allow",
                "Action": [
                    "s3:GetObject",
                    "s3:PutObject"
                ],
                "Resource": "arn:aws:s3:::*"
            }
        ]
    }''')
```

Code to create lambda:
```
lambda_py = lambda_c.create_function(
    FunctionName='TestLambda',
    Runtime='python3.7',
    Role=lambda_execution_role['Role']['Arn'],
    Handler='lambda.handler',
    Code=dict(ZipFile=zipped_code)
)
```

Code to invoke lambda:
```
event = dict(message='blah')

lambda_c.invoke(FunctionName='arn:aws:lambda:us-east-1:000000000000:function:TestLambda', InvocationType='Event',
               Payload=json.dumps(event))
```

Output from console than localstack was spawned in:
```
2019-07-10T12:08:56:INFO:root: Hello, World!
```

Getting all log groups:
```
logs_c.describe_log_groups()
```
Output:
```
{'logGroups': [],
 'ResponseMetadata': {'HTTPStatusCode': 200,
  'HTTPHeaders': {'server': 'amazon.com',
   'content-type': 'text/html; charset=utf-8',
   'content-length': '36',
   'date': 'Wed, 10 Jul 2019 12:19:30 GMT'},
  'RetryAttempts': 0}}
```

**Comment by ciaranevans** [comment]
@Black742 & @anthonydelarosa I think https://github.com/localstack/localstack/pull/1424/files may be addressing our comments... 😄

**Comment by whummer** [comment]
Thanks for reporting @Black742 & @anthonydelarosa @ciaranevans . This should be fixed in #1424 - please re-open if the problem persists.
